src/encoded/static/components/experiment.js

```diff
@@ -27,622 +27,613 @@ const anisogenicValues = [
+        if (terms && terms.length > 0) {
+            libraryPropertyString = terms.sort().join(', ');
         }
-    });
+        return { libraryPropertyString };
+    },
 
-    // Now begin the output process -- one React component per array element
-    const components = Object.keys(results).map((key) => {
-        const libraryEntry = results[key];
-        if (libraryEntry.value !== undefined || (libraryEntry.values && Object.keys(libraryEntry.values).length)) {
-            return (
-                <div key={key} data-test={libraryEntry.test}>
-                    <dt>{libraryEntry.title}</dt>
-                    <dd>
-                        {libraryEntry.value !== undefined ?
-                            <span>
-                                {(libraryEntry.component && Object.keys(libraryEntry.component).length) ?
-                                    <span>
-                                        {Object.keys(libraryEntry.component).map(componentKey => <span key={componentKey}>{libraryEntry.component[componentKey]}</span>)}
-                                    </span>
-                                :
-                                    <span>{libraryEntry.value}</span>
-                                }
-                            </span>
-                        :
-                            <span>
-                                {Object.keys(libraryEntry.values).map((replicateId) => {
-                                    const value = libraryEntry.values[replicateId];
-                                    if (libraryEntry.component && libraryEntry.component[replicateId]) {
-                                        // Display the pre-rendered component
-                                        return <span key={replicateId} className="line-item">{libraryEntry.component[replicateId]} [{replicateId}]</span>;
-                                    }
-                                    if (value) {
-                                        // Display the simple value
-                                        return <span key={replicateId} className="line-item">{value} [{replicateId}]</span>;
-                                    }
-
-                                    // No value to display; happens when at least one replicate had a value for this property, but this one doesn't
-                                    return null;
-                                })}
-                            </span>
-                        }
-                    </dd>
-                </div>
+    spikeins_used: (library) => {
+        const spikeins = library.spikeins_used;
+        let libraryPropertyString = null;
+        let libraryPropertyComponent = null;
+        if (spikeins && spikeins.length > 0) {
+            libraryPropertyString = spikeins.sort().join();
+            libraryPropertyComponent = (
+                spikeins.map((spikeinsAtId, i) => (
+                    <span key={i}>
+                        {i > 0 ? ', ' : ''}
```

Comment:
Should there be a check for `i === spikeins.length` here so there isn't a `,` at the end?

Reply:
I think this is OK. It _prepends_ a comma before the current term, so once we render the last one, we don’t render any more commas.